static/styles/night_mode.scss

```diff
@@ -236,6 +236,16 @@ on a dark background, and don't change the dark labels dark either. */
         color: hsl(0, 0%, 100%);
     }
 
+    #streamlist-toggle,
```

Comment:
The colors of the icons should be changed too. Ideally, something like below:
![screenshot at aug 14 16-00-43](https://user-images.githubusercontent.com/15116870/44180290-61c83880-a0b0-11e8-8d27-65dc9459b5ae.png)

Reply:
@synicalsyntax OK, I've updated the color of the two icons to be the same as the home icon (`#dddeee`). I also edited the initial PR comment with a new screenshot showing this change. Please let me know if there is anything else that I should change or fix. Thanks!

Reply:
I don't see a double border with the following git diff, but correct me if I'm wrong:

```diff
diff --git a/static/styles/night_mode.scss b/static/styles/night_mode.scss
index f8263670d..2f7b09c49 100644
--- a/static/styles/night_mode.scss
+++ b/static/styles/night_mode.scss
@@ -76,6 +76,8 @@ on a dark background, and don't change the dark labels dark either. */
 
     .new-style .button:not(.save-status),
     .new-style .button[disabled=disabled]:hover,
+    #streamlist-toggle-button,
+    #userlist-toggle-button,
     input[type="text"],
     input[type="email"],
     input[type="password"],
diff --git a/static/styles/right-sidebar.scss b/static/styles/right-sidebar.scssindex 6c384f8b9..5b9c2fd8b 100644
--- a/static/styles/right-sidebar.scss
+++ b/static/styles/right-sidebar.scss
@@ -186,12 +186,12 @@
     right: 0px;
     text-align: center;
     vertical-align: middle;
-    border-left: 2px solid hsl(204, 20%, 74%);
 }
 
 #userlist-toggle-button {
     text-decoration: none;
     background-color: hsl(0, 0%, 89%);
+    border-left: 2px solid hsl(204, 20%, 74%);
     color: hsl(0, 0%, 52%);
     display: block;
     width: 40px;
diff --git a/static/styles/zulip.scss b/static/styles/zulip.scss
index 4178fe3f3..ba5464333 100644
--- a/static/styles/zulip.scss
+++ b/static/styles/zulip.scss
@@ -1689,7 +1689,6 @@ blockquote p {
     left: 0px;
     text-align: center;
     vertical-align: middle;
-    border-right: 2px solid hsl(204, 20%, 74%);
 }
 
 #streamlist-toggle-button {
@@ -1698,6 +1697,7 @@ blockquote p {
     display: block;
     position: relative;
     background-color: hsl(0, 0%, 89%);
+    border-right: 2px solid hsl(204, 20%, 74%);
     width: 40px;
     height: 19px;
     padding-top: 12px;
(END)
```
![screenshot at aug 14 16-00-43](https://user-images.githubusercontent.com/15116870/44232449-ee790200-a155-11e8-9916-82c104d8ad3f.png)